docassemble/ALAffidavitOfIndigency/data/questions/affidavit_of_indigency_supplement_body.yml

```
@@ -691,13 +691,16 @@ fields:
   - "What is the value of the home?": user_home_value
     show if: user_owns_home
     datatype: currency
+    required: True
     min: 0
   - Are there any remaining loans on the home?: user_has_home_debt
     show if: user_owns_home
     datatype: yesnoradio
+    required: True
   - How much do you have to pay on the remaining loans?: user_home_debt
     show if: user_has_home_debt
     datatype: currency
+    required: True
```

```suggestion
    min: 0
  - Are there any remaining loans on the home?: user_has_home_debt
    show if: user_owns_home
    datatype: yesnoradio
  - How much do you have to pay on the remaining loans?: user_home_debt
    show if: user_has_home_debt
    datatype: currency
```

`required: True` is default so probably best not to clutter here